std/math/fp2.asm

```diff
@@ -20,9 +20,8 @@ use std::prover::eval;
 /// f = x^2 - 7
```

Comment:
Outdated comment. I guess the X^2 - 7 is not irreducible in BabyBear?

Reply:
- fp2 is a quadratic tower extension of a field element. We use the fp2 extension field element (instead of ff.asm) in permutation/lookup arguments in PIL to increase the security of bits. For example, if we use Goldilocks field (2^64) on permutation arguments in PIL, we could have 64-18 = 46 bits of estimated security for the 2^18 number of rows. Using fp2 = (fp, fp) basically. We know have 64+64-18 = 110 bits of estimated security

- Yes, x^2 - 7 is not irreducible, we should've have x^2 - 11, thanks for the catch